src/test/java/uk/gov/digital/ho/pttg/SimpleHttpClient.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.digital.ho.pttg;
+
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import uk.gov.digital.ho.pttg.api.HttpResponse;
+
+import java.io.IOException;
+
+class SimpleHttpClient {
+
+    HttpResponse post(String url, String body) {
+        try {
+            return doPost(url, body);
```

Comment:
I don't but I didn't like having nested try blocks - this keeps the handling of IOExceptions separate from ensuring that the closing of the client.

I can't do
```
try{
    request....
} catch(IOException e) {
    //handle it
} finally {
    httpClient.close()
}
```
because the `httpClient.close()` throws an IO exception - leading to nested try blocks.